lib/gilmour/backends/redis.rb

```diff
@@ -42,8 +45,8 @@ def setup_pubsub(opts)
       @subscriber = @publisher.pubsub_client
       register_handlers
     rescue Exception => e
-      GLogger.debug e.message
```

Comment:
I think these should all remain debug. These will be required only for debugging gilmour itself and can be enabled by setting `LOGGER_LEVEL=debug` environment variable when required.
Only those errors pertaining to the result of the handlers should be error and even those should go on the logger of the responder object.